lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -0,0 +1,128 @@
+"""
+This is still an experimental module and there will almost certainly be backward
+incompatible changes coming.
+"""
+
+
+import os
+
+from ..resolvers import DependencyResolver, INDETERMINATE_DEPENDENCY
+from ..conda_util import (
+    CondaContext,
+    CondaTarget,
+    install_conda,
+    is_conda_target_installed,
+    install_conda_target,
+    build_isolated_environment,
+)
+
+DEFAULT_ENSURE_CHANNELS = "r,bioconda"
```

Comment:
It is configurable, if you follow the tracks I think conda_ensure_channels will cause the dependency resolver to pick these up. I guess I should update galaxy.ini.sample with these options.